supervisor/shared/serial.c

```diff
@@ -107,6 +107,29 @@ int console_uart_printf(const char *fmt, ...) {
     #endif
 }
 
+MP_WEAK void board_serial_early_init(void) {
+}
+
+MP_WEAK void board_serial_init(void) {
+}
+
+MP_WEAK bool board_serial_connected(void) {
+    return false;
+}
+
+MP_WEAK char board_serial_read(void) {
+    return -1;
+}
+
+MP_WEAK uint32_t board_serial_bytes_available(void) {
+    return false;
+}
+
```

Comment:
Should this be
    return 0;

Reply:
Due to #9141 being recently merged, yes, this should be `0`.